docs-website/.astro/content.d.ts

```
@@ -0,0 +1,158 @@
+  // Invalid collection names will be caught at build time.
+  export function reference<C extends string>(
+    collection: C
+  ): import('astro/zod').ZodEffects<import('astro/zod').ZodString, never>;
+
+  type ReturnTypeOrOriginal<T> = T extends (...args: any[]) => infer R ? R : T;
+  type InferEntrySchema<C extends keyof AnyEntryMap> =
+    import('astro/zod').infer<
+      ReturnTypeOrOriginal<Required<ContentConfig['collections'][C]>['schema']>
+    >;
+
+  type ContentEntryMap = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)